src/components/Login/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { addUser } from '../../store/actions';
+
+class Login extends Component {
+	state = {
+		username: '',
+		password: '',
+	};
+
+	handleChanges = event => {
+		this.setState({ [event.target.name]: event.target.value });
+	};
+
+	setUserOnStorage = event => {
```

Comment:
This is where you'll eventually put the POST request to the server to login I assume?